tests/unit/libbpf_test.cpp

```diff
@@ -3233,14 +3233,68 @@ TEST_CASE("bind_tail_call_max_exceed", "[libbpf]")
     usersim_trace_logging_set_enabled(false, 0, 0);
 }
 
-TEST_CASE("libbpf map batch", "[libbpf]")
+void
+_test_batch_iteration_maps(fd_t& map_fd, uint32_t batch_size, bpf_map_batch_opts* opts, bool concurrent_delete)
+{
+    // Retrieve in batches.
+    uint32_t* next_key_ptr = nullptr;
+    uint32_t next_key = 0;
+    std::vector<uint32_t> returned_keys(0);
+    std::vector<uint64_t> returned_values(0);
+    uint32_t requested_batch_size = batch_size / 10 - 2;
+    uint32_t temp_batch_size = 0;
+    int ret = 0;
```

Comment:
https://github.com/microsoft/ebpf-for-windows/blob/main/docs/DevelopmentGuide.md says
> **DON'T** use abbreviations unless they are already well-known terms known by users (e.g., "app", "info"), or are already required for use by developers (e.g., "min", "max", "args"). Examples of bad use would be num_widgets instead of widget_count, and opt_widgets instead of option_widgets or optional_widgets.

"ptr", "temp", and especially "ret" all arguably violate the ebpf-for-windows coding conventions.

Reply:
Thanks. Fixed as per the guidelines. Please check.